app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessBusiness.java

```diff
@@ -36,13 +41,25 @@
 				"courthouse", "embassy", "car_wash", "car_rental",
 				"marketplace", "fuel", "driving_school", "dentist",
 				"doctors", "clinic", "pharmacy", "veterinary",
-				"place_of_worship", "townhall"};
+				"place_of_worship", "townhall", "school", "kindergarten",
+				"shelter", "hospital", "police", "community_centre",
+				"social_facility", "public_building", "university", "college",
+				"theatre", "arts_centre", "bbq", "nursing_home",
+				"ferry_terminal", "childcare", "shower", "prison", "studio",
+				"casino", "internet_cafe", "public_bookcase", "brothel",
+				"dojo", "gambling", "payment_terminal", "events_venue",
+				"public_bath", "training", "social_centre", "animal_breeding",
+				"animal_shelter", "retirement_home", "monastery", "sauna",
+				"lavoir", "music_school", "animal_boarding", "ranger_station",
+				"love_hotel", "crematorium", "gym", "stables", "stripclub",
+				"dancing_school", "language_school", "photo_booth", "mortuary",
+				"coworking_space", "customs", "conference_centre",
+				"smoking_area", "payment_centre", "research_institute" };
 
 		String[] tourism = {
-				"zoo", "aquarium", "theme_park", "gallery",
-				"museum", "hotel", "guest_house", "hostel",
-				"motel", "viewpoint"
-		};
+				"zoo", "aquarium", "theme_park", "gallery", "museum", "hotel",
+				"guest_house", "hostel", "motel", "viewpoint", "attraction",
+				"chalet", "apartment", "resort" };
```

Comment:
hard to compare, as you reformated the value. I am undecided with attraction, but looks good otherwise

Reply:
apartment is unlikely to be easily surveyable

Reply:
@HolgerJeromin yeah, sorry for this. But I tried to give those array the same width as the others before.

@matkoniecz yes, some of them may sometimes be a bit harder, but this really depends on the facility itself and cannot be applied to all apartments. Keep in mind, that those have probably a web page or information flyers or staff you could ask, without renting them.

Reply:
> Keep in mind, that those have probably a web page or information flyers or staff you could ask, without renting them.

this kind of Survey is not the way this app should work.